### PR TITLE
Add an icon to open service test doc

### PR DESCRIPTION
**Pull request by YannanGao-gs**
## Summary

Fixes https://github.com/finos/legend-studio/issues/1621

## How did you test this change?

- [ ] Test(s) added
- [x] Manual testing (please provide screenshots/recordings)
- [ ] No testing (please provide an explanation)

https://user-images.githubusercontent.com/73408381/200399013-ab69af62-7670-44e9-ad8c-bd9f037248da.mov

### CODE REVIEW

File: packages/legend-application-studio/src/components/editor/edit-panel/service-editor/ServiceEditor.tsx [hunk 1]
```diff
@@ -445,6 +454,16 @@ export const ServiceEditor = observer(() => {
                   })}
                 >
                   {prettyCONSTName(tab)}
+                  {tab === SERVICE_TAB.TEST && (
+                    <button
+                      className="service-editor__tab__hint"
+                      tabIndex={-1}
+                      onClick={seeDocumentation}
+                      title="Click to see more details on advanced search"
```
Comment (xannem) [inline packages/legend-application-studio/src/components/editor/edit-panel/service-editor/ServiceEditor.tsx:462]: Maybe like click to see more details on service test?

Comment (YannanGao-gs) [reply packages/legend-application-studio/src/components/editor/edit-panel/service-editor/ServiceEditor.tsx:462]: hhahah yes copy and paste

File: packages/legend-application-studio/style/components/editor/_service-editor.scss [hunk 2]
```diff
@@ -103,6 +103,18 @@
     background: var(--color-yellow-200);
   }
 
+  &__tab__hint {
+    @include flexCenter;
+
+    border-radius: 0.2rem;
+    margin-left: 1rem;
```
Comment (xannem) [inline packages/legend-application-studio/style/components/editor/_service-editor.scss:110]: very minor, I think 0.5rem might be better for spacing than 1rem, like how info icon for text-search-advanced-config__panel__header__hint has 0.5 padding. But that is subjective I think ^^

**Comment by xannem** [comment]
@MauricioUyaguari review done 🙇

**Comment by MauricioUyaguari** [comment]
Can we add an icon next to "connection test data" and add entries for defining test data. 
I think this also requires enhancing the docs. 
Something like this
"How do I specify relational test data?"
"How do I share data across tests?"
"How do I specify service store data?"
"How do I specify data for model to model execution?"

**Review by MauricioUyaguari — APPROVED** [review]

**Pull request merged**